pkg/commands/logs.go

```diff
@@ -69,7 +69,7 @@ func Run(ctx context.Context, config *stern.Config, ioStreams cmdutil.IOStreams)
 			case str := <-logC:
 				ioStreams.Infonln(str)
 			case <-ctx.Done():
-				break
+				return
```

Comment:
Good catch!

Reply:
Consider adding some golang static inspection tools to github actions.